analytics/lib/counts.py

```diff
@@ -79,7 +79,8 @@ def process_count_stat(stat: CountStat, fill_to_time: datetime) -> None:
 
     verify_UTC(fill_to_time)
     if floor_to_hour(fill_to_time) != fill_to_time:
-        raise ValueError("fill_to_time must be on an hour boundary: %s" % (fill_to_time,))
+        raise ValueError(
+            "fill_to_time must be on an hour boundary: %s" % (fill_to_time,))
```

Comment:
Can you remove these spurious reformattings?  They make reading your changes difficult.